swagger-coverage-commandline/src/main/java/com/github/viclovsky/swagger/coverage/core/results/builder/prebuilder/CoverageStatisticsBuilder.java

```diff
@@ -49,6 +51,22 @@ public CoverageStatisticsBuilder add(OpenAPI swagger) {
                     .filter(equalsOperationKeys(key)).findFirst();
 
             if (keyOptional.isPresent()) {
+                // Recreate path parameters if framework can't parse path parameters to value
+                Map<String, String> extractedPathParams = new AntPathMatcher()
+                        .extractUriTemplateVariables(keyOptional.get().getPath(), key.getPath());
+                if (!extractedPathParams.isEmpty()) {
+                    List<String> existedPathParametersNames = value.getParameters().stream()
```

Comment:
I've tried your fix, and it works great! However, in my case, I encountered a NullPointerException on `value.getParameters().stream()`.

Perhaps we could handle it like this:
```                    
List<Parameter> existedPathParameters = (value.getParameters() != null) 
    ? value.getParameters() 
    : new ArrayList<>();

List<String> existedPathParametersNames = existedPathParameters.stream()
    .filter(parameter -> parameter.getIn().equals("path"))
    .map(parameter -> parameter.getName())
    .collect(Collectors.toList());
```
Looking forward to seeing your PR accepted soon!